ntoskrnl/mm/ARM3/pool.c

```diff
@@ -1604,8 +1608,7 @@ MmAllocateMappingAddress(
     }
 
     ASSERT(SizeInPages <= MM_EMPTY_PTE_LIST);
-    TempPte.u.Long = 0;
-    TempPte.u.List.NextEntry = SizeInPages;
+    TempPte.u.Long = SizeInPages << 1;
```

Comment:
Please revert

Reply:
Reverted. Now we have 47 failed tests.

![w2k3sp2_kmtest_MmReservedMapping_fails](https://github.com/user-attachments/assets/922407fa-007b-413e-b596-8ac19059275c)

Can you please help me with fixing these new failing tests?